xslts/constraints-split-use-access.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+Sépare les conditions d'utilisation des conditions d'accès.
+
+
+## Paramètres
+
+Aucun.
+
+
+## Motivation
+
+Lorsqu'un élément `gmd:resourceConstraints` contient à la fois un élément `gmd:accessConstraints` et un élément `gmd:useConstraints`, tous les éléments `gmd:otherConstraints` également présents vont être doublonnés en condition d'utilisation *et* en condition d'accès par le convertisseur SEMICeu ISO-19139 vers GeoDCAT-AP.
+
+Un tel cas ne respectant pas le standard INSPIRE, il est difficile motiver une évolution du convertisseur SEMICeu.
+
+
+## Limitations
+
+La transformation prend en charge uniquement les cas pour lesquels la séparation des conditions d'utilisation et d'accès : 
+
```

Comment:
It's the markdown flavor that requires a space before a list... Is that tweakable somewhere?

Reply:
Fixed by https://github.com/ecolabdata/ecospheres-isomorphe/pull/102